Workbooks/Azure Private MEC/PMEC Site Overview/PMEC Site Overview.workbook

```diff
@@ -1818,7 +1818,78 @@
             "type": 3,
             "content": {
               "version": "KqlItem/1.0",
-              "query": "alertsmanagementresources\r\n| where type == 'microsoft.alertsmanagement/alerts'\r\n| extend severity = tostring(properties[\"essentials\"][\"severity\"])\r\n| extend startTime = properties[\"essentials\"][\"startDateTime\"]\r\n| extend description = properties[\"essentials\"][\"description\"]\r\n| extend condition = properties[\"essentials\"][\"monitorCondition\"]\r\n| where properties[\"essentials\"][\"targetResource\"] =~ '{ase}'\r\n| project [\"Alert Time\"]=todatetime(startTime),[\"Severity\"]=severity,[\"Alert\"]=name,[\"Description\"]=description,[\"Alert Condition\"]=condition\r\n| order by [\"Alert Time\"] desc",
+              "query": "{\"version\":\"ARMEndpoint/1.0\",\"data\":null,\"headers\":[],\"method\":\"GET\",\"path\":\"{ase}/networkSettings/default?api-version=2021-02-01\",\"urlParams\":[{\"key\":\"\",\"value\":\"\"},{\"key\":\"\",\"value\":\"\"}],\"batchDisabled\":false,\"transformers\":[{\"type\":\"jsonpath\",\"settings\":{\"tablePath\":\"properties.networkAdapters\",\"columns\":[{\"path\":\"adapterPosition.port\",\"columnid\":\"Port\",\"columnType\":\"string\"},{\"path\":\"ipv4Configuration.ipAddress\",\"columnid\":\"IP\",\"columnType\":\"string\"},{\"path\":\"ipv4Configuration.gateway\",\"columnid\":\"Gateway\",\"columnType\":\"string\"},{\"path\":\"ipv4Configuration.subnet\",\"columnid\":\"Subnet\",\"columnType\":\"string\"},{\"path\":\"dhcpStatus\",\"columnid\":\"DHCP\",\"columnType\":\"string\"}]}}]}",
```

Comment:
It would be more accurate to make the title of the column "Subnet mask" here rather than "Subnet".